README.md

```diff
@@ -236,12 +240,12 @@ View the docs by pointing your browser at: `…/rust-fil-proofs/target/doc/proof
 
 ## API Reference
 
-The **FPS** is accessed from [**go-filecoin**](https://github.com/filecoin-project/go-filecoin) via FFI calls to its API, which is the union of the APIs of its constituents:
+The **FPS** is accessed from [**lotus**](https://github.com/filecoin-project/lotus) via FFI calls to its API, which is the union of the APIs of its constituents:
 
  The Rust source code serves as the source of truth defining the **FPS** APIs. View the source directly:
 
-- [**filecoin-proofs**](https://github.com/filecoin-project/rust-fil-proofs/blob/master/filecoin-proofs/src/api/mod.rs)
-- [**filecoin-proofs-api**](https://github.com/filecoin-project/rust-filecoin-proofs-api).
+- [**filecoin-proofs**](https://github.com/filecoin-project/rust-fil-proofs/tree/master/filecoin-proofs/src/api)
+- [**filecoin-proofs-api**](https://github.com/filecoin-project/rust-filecoin-proofs-api)
```

Comment:
This probably bears some explanation. `rust-filecoin-proofs-api` now *is* the API, and what it provides (rather than the API implementations in `rust-fil-proofs`) should probably be what most consumers consider. 

I am not sure how much detail we want/need to go into here/now, but if we are going to mention a specific consumer, we probably need to at least clarify that no one should be using `rust-fil-proofs` directly. The versioned API in `rust-filecoin-proofs-api` is the entry point all consumers should use; and this itself should only be used by those who have some need to bypass the Go API wrapper provided in `filecoin-ffi`.